improver/blending/utilities.py

```diff
@@ -175,10 +175,12 @@ def _set_blended_time_coords(blended_cube: Cube, cycletime: Optional[str]) -> No
     """
     For cycle and model blending:
     - Add a "blend_time" coordinate equal to the current cycletime
-    - Update the forecast reference time and forecast period coordinate points
-    to reflect the current cycle time (behaviour is DEPRECATED)
+    - Update the forecast reference time to reflect the current cycle time
+      (behaviour is DEPRECATED)
+    - If a forecast period coordinate is present this will be updated relative
+      to the current cycle time (behaviour is DEPRECATED)
```

Comment:
I think a few of these lines need to be altered to comply with [Sphinx formatting](https://github.com/metoppv/improver/runs/8034112451?check_suite_focus=true). Perhaps this?

```suggestion
    - Update the forecast reference time to reflect the current cycle time
    (behaviour is DEPRECATED)
    - If a forecast period coordinate is present this will be updated relative
    to the current cycle time (behaviour is DEPRECATED)
```

Reply:
Good spot. Fixed. FYI your suggested fix would not produce a bulleted list in the rendered output. What's needed here is a blank line between the line before the list ending in a colon and the bulleted list itself.